vlingo-reactive-messaging-patterns/src/main/java/io/vlingo/reactive/messaging/patterns/scattergather/BestPriceQuotation.java

```diff
@@ -0,0 +1,104 @@
+// was not distributed with this file, You can obtain
+// one at https://mozilla.org/MPL/2.0/.
+package io.vlingo.reactive.messaging.patterns.scattergather;
+
+import java.util.Vector;
+
+/**
+ * BestPriceQuotation
+ *
+ * @author brsg.io
+ * @since Nov 21, 2018
+ */
+public class BestPriceQuotation
+{
+    public final String rfqId;
+    public final Vector<PriceQuote> priceQuotes;
+    
+    public BestPriceQuotation( String rfqId, PriceQuote...quotes )
+    {
+        this.rfqId = rfqId;
+        this.priceQuotes = new Vector<>();
+        for ( PriceQuote quote : quotes )
+        {
+            this.priceQuotes.add( quote );
+        }
+    }
+
+    /* @see java.lang.Object#toString() */
+    @Override
+    public String toString()
+    {
+        StringBuilder sb = new StringBuilder();
+        sb.append( String.format( "%s: ", rfqId ));
+        Object[] quotes = null;
+        if ( priceQuotes.size() > 0 )
+        {
+            for ( int i = 0; i < priceQuotes.size(); i++ )
+            {
+                sb.append( " (%s) " );
+            }
+            quotes = priceQuotes.toArray( new PriceQuote[ priceQuotes.size() ]);
+        }
+        return priceQuotes.size() > 0 ? String.format( sb.toString(), quotes ) : sb.toString();
+    }
+    
+    public static class PriceQuote
+    {
+        public final QuoteProcessor quoteProcessor;
+        public final String rfqId;
+        public final String itemId;
+        public final Double retailPrice;
+        public final Double discountPrice;
+        
+        public PriceQuote( final QuoteProcessor quoteProcessor, final String rfqId, final String itemId, final Double retailPrice, final Double discountPrice )
+        {
+            this.quoteProcessor = quoteProcessor;
+            this.rfqId = rfqId;
+            this.itemId = itemId;
+            this.retailPrice = retailPrice;
+            this.discountPrice = discountPrice;
+        }
+
+        /* @see java.lang.Object#toString() */
+        @Override
+        public String toString()
+        {
+            return String.format( "%s %s %s $%.2f $%.2f", quoteProcessor.getClass().getSimpleName(), rfqId, itemId, retailPrice, discountPrice );
+        }
+    }
+    
+    public static class QuotationFulfillment
+    {
+        public final String rfqId;
+        public final Integer quotesRequested;
+        public final Vector<PriceQuote> priceQuotes;
+        public final OrderProcessor requestor;
+        
+        public QuotationFulfillment( final String rfqId, Integer quotesRequested, OrderProcessor requestor, PriceQuote...quotes )
+        {
+            this.rfqId = rfqId;
+            this.quotesRequested = quotesRequested;
+            this.requestor = requestor;
+            this.priceQuotes = new Vector<>();
+            for ( PriceQuote quote : quotes )
+            {
+                priceQuotes.add( quote );
+            }
+        }
+    }
+    
+    public static class PriceQuoteTimedOut
```

Comment:
Is used in PriceQuoteAggregator.

Reply:
Hmmm, ok, it's not in this commit. Is it in a commit with a different examples project? 

I just commented on this in the slack channel noting that this can cause issues, such as in reviews. No big deal, but try to look out for these cross-project commits.